starter-code/javascript/chronometer.js

```diff
@@ -1,10 +1,51 @@
 class Chronometer {
-  constructor() {}
-  // startClick() {}
-  // getMinutes() {}
-  // getSeconds() {}
-  // twoDigitsNumber() {}
-  // stopClick() {}
-  // resetClick() {}
-  // splitClick() {}
+  constructor() {
+    this.currentTime = 0,
+    this.intervalId
+    this.savedTime = []
+  }
+   startClick() {
+    this.intervalId = setInterval(() => {
+      this.currentTime ++
+      printTime()
+    },1000)
+  }
+  getMinutes() {
+     return Math.floor(this.currentTime/60)
+  }
+  getSeconds() {
+      return Math.floor(this.currentTime%60)
+  }
+  twoDigitsNumber(time) {
+    //  const twoDigit = ''
+    //  if(time < 10) {
+    //   twoDigit = `0${time}`
+    //  } else {
+    //   twoDigit = `${time}`
+    //  }
+    //  return twoDigit
+     return (time < 10 ? `0${time}` : `${time}`)
+  }
+   stopClick() {
+    clearInterval(this.intervalId)
+   }
+
+   resetClick() {
```

Comment:
esta funcion tiene codigo de mas, 
con que pongan: 
`  resetClick() {`
   ` this.currentTime = 0`
`  }`

funciona y jasmine les da verde :)